src/tooltip.js

```diff
@@ -32,16 +29,21 @@
         }
         */
         var data = null;
-
         var gravity = 'w'   //Can be 'n','s','e','w'. Determines how tooltip is positioned.
-            ,distance = 50   //Distance to offset tooltip from the mouse location.
-            ,snapDistance = 25   //Tolerance allowed before tooltip is moved from its current position (creates 'snapping' effect)
+            ,   distance = 25   //Distance to offset tooltip from the mouse location.
+            ,   snapDistance = 0   //Tolerance allowed before tooltip is moved from its current position (creates 'snapping' effect)
             ,   fixedTop = null //If not null, this fixes the top position of the tooltip.
             ,   classes = null  //Attaches additional CSS classes to the tooltip DIV that is created.
-            ,   chartContainer = null   //Parent DIV, of the SVG Container that holds the chart.
+            ,   chartContainer = null   //Parent dom element of the SVG that holds the chart.
+            ,   hidden = true  // start off hidden, toggle with hide/show functions below
+            ,   hideDelay = 400  // delay before the tooltip hides after calling hide()
```

Comment:
linePlusBarChart was the worst, there are spaces between the bars that cause it.  However, even the line charts were jumpy if you didn't take care to go from one point to another or the points were far enough apart.  

I had to do a similar thing a long time ago when crating a tooltip over jqplot graphs... due to tiny gaps here and there it's a lot safer to keep a small delay there for hiding.  I bumped up the default a bit since I thought it looked smoother, if you like it hiding faster we can bump it down some though, we just can't really remove it entirely.